<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClient</class>
 <widget class="QMainWindow" name="GUIClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GUIClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <!--widget class="QWidget" name="T0_Tab">
     <attribute name="title">
      <string>T0_Correlation</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="T0_layout">
       <item>
        <widget class="QPushButton" name="GetT0_Correlation">
         <property name="text">
          <string>Get T0 Correlation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GetT0s_Difference">
         <property name="text">
          <string>Get T0 Difference</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GetT0s_Sum">
         <property name="text">
          <string>Get T0 Sum</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget-->
    <widget class="QWidget" name="Hits_Tab">
     <attribute name="title">
      <string>Hits</string>
     </attribute>
     <widget class="QPushButton" name="GetHits">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>150</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get NHits Profiles</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GetHits_perLayer">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>200</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get NHits per layer (0.5 MIP)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GetnHits_Per_Event">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>150</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get nHits Per Event</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GetMemoryCells">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>150</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Memory Cells</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="EnergySum_Tab">
     <attribute name="title">
      <string>Energy</string>
     </attribute>
     <widget class="QPushButton" name="GetEnergySum">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Energy Sum</string>
      </property>
     </widget>
     <!--widget class="QPushButton" name="GetEnergyPerLayer">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Energy Per Layer</string>
      </property>
     </widget-->
     <widget class="QPushButton" name="GetEnergyPerChannel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Energy Per Channel</string>
      </property>
     </widget>
    <widget class="QPushButton" name="GetRMSPerChannel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get RMS Per Channel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HitTime_Tab">
     <attribute name="title">
      <string>Time</string>
     </attribute>
     <widget class="QPushButton" name="GetHitTime">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>171</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Hit Time</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Shower_Tab">
     <attribute name="title">
      <string>Shower Variables</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>409</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="GetShower">
         <property name="text">
          <string>Get Shower Variables</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GetEnergyLayer">
         <property name="text">
          <string>Energy Per Layer</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GetShower_others">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>More plots</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="HitMapTab">
     <attribute name="title">
      <string>HitMaps Layers</string>
     </attribute>
     <widget class="QPushButton" name="GetHitMap">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get HitMaps</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    <widget class="QPushButton" name="GetHitMap_Log">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get HitMaps_Log</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="GetHitMap_Energy">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get HitMaps_Energy</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="nHitscogZ_Tab">
     <attribute name="title">
      <string>nHitscogZ</string>
     </attribute>
     <widget class="QPushButton" name="GetnHitscogZ">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>171</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string>Get nHits_vs_cogZ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TempTab">
     <attribute name="title">
      <string>Temperature</string>
     </attribute>
     <widget class="QPushButton" name="GetTemp">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Temperature Histo</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="LayoutStatusServer">
     <item>
      <widget class="QLabel" name="Label_status">
       <property name="text">
        <string>Status Server :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Show_status">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>301</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ServerIPLayout">
     <item>
      <widget class="QLabel" name="label_server">
       <property name="text">
        <string>Server IP :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line_IP">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Connect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Logger" name="LoggerWid" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>771</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="RunNumbe_layout">
     <item>
      <layout class="QHBoxLayout" name="RunLayout">
       <item>
        <widget class="QLabel" name="Run_Label">
         <property name="text">
          <string>Run Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RunNumber">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Exit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Logger</class>
   <extends>QWidget</extends>
   <header location="global">Logger.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
